modules/tooltip/css/stories.scss

```diff
@@ -0,0 +1,13 @@
+.tooltip-demo {
```

Comment:
Can you delete this file and just use emotion in the story file for this margin?